partner_event/models/res_partner.py

```diff
@@ -20,15 +20,16 @@ class ResPartner(models.Model):
         help="Count of events with confirmed registrations.",
     )
     registration_count = fields.Integer(
-        string='Event registrations number', compute='_compute_registrations',
+        string='Event registrations number', 
+	compute='_compute_count_registration',
         store=True)
     attended_registration_count = fields.Integer(
         string='Event attended registrations number',
         compute='_compute_attended_registrations', store=True)
 
     @api.multi
     @api.depends('registrations')
-    def _compute_registrations(self):
+    def _compute_count_registration(self):
```

Comment:
I think @dreispt means that the compute name should [follow the guidelines](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#models): `_compute_<fieldname>`.